code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -272,6 +272,10 @@ meteor_act
 	if(!affecting)
 		return FALSE//should be prevented by attacked_with_item() but for sanity.
 
+	if(user.a_intent == I_HELP)
+		visible_message("<span class='danger'>[src] has been [pick("lightly poked", "tapped")] in the [affecting.name] with [I.name] by [user]!</span>")
```

Comment:
We have contribution guidelines, try reading them, staff developer.
```suggestion
		visible_message(SPAN_DANGER("[src] has been [pick("lightly poked", "tapped")] in the [affecting.name] with [I.name] by [user]!"))
```

Reply:
Sigmar guides my code, not some feeble guidelines